packages/internet-header/src/components/post-internet-breadcrumbs/post-internet-breadcrumbs.scss

```diff
@@ -18,6 +19,7 @@
   align-items: center;
   justify-content: space-between;
   gap: 1rem;
+  min-height: spacing.$size-small-huge; // Avoid visual shift when buttons are hidden.
```

Comment:
Maybe there is a cleaner way…

Reply:
Not sure if this is really needed? DEVs will probably not toggeling these buttons during a user visit.
Instead they will decide, that in their specific project, those buttons are not needed at all or only on some pages.

Or at least we could use the button height variable `$btn-height-md` (3rem) from styles/src/variables/components/_button.scss.

Reply:
I agree with the fact that they will not be toggling between those two states, but I think it should be the same size across projects for homogeneity and to make it easier to reason about with mockup.

Reply:
Yep it's done